docs: Add io.cozy.coachco2.settings (#245)
cozy/cozy-doctypes: docs/io.cozy.coachco2.md

# Cozy CoachCO2 settings doctype

> This doctype store information about CoachCO2 application settings. There is only one document.

Reviewer: ```suggestion
This doctype stores information about CoachCO2 application settings. There is only one document.
```

The doctype stores the following data:

> - `account`: {Object} - Relative account, copy of `io.cozy.account` document

Reviewer: ```suggestion
- `account`: {Object} - Current account displayed, copy of the associated `io.cozy.account` document
```

- `bikeGoal`: {Object} - Informations about the bike goal feature
  - `showAlert`: {boolean} - Either an alert is displayed requesting activation of the bike goal feature
  - `showAlertSuccess`: {boolean} - Either an alert is displayed when the goal is reached
  - `activated`: {boolean} - Either the feature is enabled
  - `sendToDACC`: {boolean} - Either the user has agreed to share their data with the DACC

>   - `onboarded`: {boolean} - Either the user as completed the onboarding

Reviewer: I don't think "either" is grammatically correct here, it's more "If" or "Whether" :)

Author: Whether is fine, thanks

  - `onboardingStep`: {number} - The actual onboard step of the user
  - `firstname`: {string} - The user's firstname requested in the onboarding
  - `lastname`: {string} - The user's lastname requested in the onboarding
  - `daysToReach`: {number} - The user's goal days to reach requested in the onboarding

> - `CO2Emission`: {Object} - Informations abount the CO2 Emission feature

Reviewer: ```suggestion
- `CO2Emission`: {Object} - Informations about the CO2 Emission feature
```
